R/communicate.R

```diff
@@ -0,0 +1,27 @@
+#' Communiate a warning if necessary
+#'
+#' If roundtrip verification was not possible, issue a warning to review the
+#' changes carefully.
+#' @param changed Boolean with indicating for each file whether or not it has
+#'   been changed.
+#' @inheritParams can_verify_roundtrip
+communicate_warning <- function(changed, transformers) {
+  if (any(changed, na.rm = TRUE) & !can_verify_roundtrip(transformers)) {
+    cat("Please review the changes carefully!")
```

Comment:
Is it ok to use `cat()` or do you prefer `warning()`. I just felt red is not the right color.

Reply:
We can add color with crayon if necessary.